### PR TITLE
Weblate translations

### PR DESCRIPTION
**Pull request by renatolond**

### CODE REVIEW

File: config/locales/ca.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ ca:
     nothing_here: No hi ha res aquí!
     people_followed_by: Usuaris a qui %{name} segueix
     people_who_follow: Usuaris que segueixen %{name}
-    posts: Toots
+    posts: ''
```
Comment (ykzts) [inline config/locales/ca.yml]: empty?

File: config/locales/ca.yml [hunk 2]
```diff
@@ -330,17 +330,20 @@ ca:
       confirmed: Confirmat
       expires_in: Expira en
       last_delivery: Últim lliurament
-      title: WebSub
+      title: ''
```
Comment (ykzts) [inline config/locales/ca.yml]: Empty?

File: config/locales/ca.yml [hunk 3]
```diff
@@ -170,7 +170,7 @@ ca:
       destroyed_msg: Emojo s'ha destruït amb èxit!
       disable: Inhabilita
       disabled_msg: S'ha inhabilitat l'emoji amb èxit
-      emoji: Emoji
+      emoji: ''
```
Comment (ykzts) [inline config/locales/ca.yml]: Empty?

File: config/locales/ca.yml [hunk 4]
```diff
@@ -518,7 +530,7 @@ ca:
   pagination:
     next: Següent
     prev: Enrere
-    truncate: "&hellip;"
+    truncate: ''
```
Comment (ykzts) [inline config/locales/ca.yml]: Empty?

File: config/locales/ca.yml [hunk 5]
```diff
@@ -547,43 +559,43 @@ ca:
...
       otter: Altre
-      phantom_js: PhantomJS
+      phantom_js: ''
       qq: QQ Browser
-      safari: Safari
-      uc_browser: UCBrowser
-      weibo: Weibo
+      safari: ''
+      uc_browser: ''
+      weibo: ''
```
Comment (ykzts) [inline config/locales/ca.yml]: umm...

**Comment by ykzts** [comment]
@renatolond I think that you should restore the place changed to `''` of [config/locales/ca.yml](https://github.com/tootsuite/mastodon/pull/6323/files#diff-9f87c1ff5aec9eb5e604c04f8e07089b). 

cc @salvadorpla

File: config/locales/ca.yml (hunk 5 above)
Comment (renatolond) [reply config/locales/ca.yml]: Added them back, new weblate weirdness, yay 🙃

**Review by Gargron — APPROVED** [review]

**Pull request merged**